.travis.yml

```diff
@@ -66,6 +66,7 @@ cache:
 # end_cache_directories
   - packages/java-debug/download
   - packages/debug-nodejs/download
+  - /tmp/vscode-ripgrep-cache-1.2.4
```

Comment:
only concern that we should keep it sync with rigrep version, maybe we can improve `scripts/prepare-travis`

Reply:
I will have a try

Reply:
done